external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/ConsumerStrategy.scala

```diff
@@ -54,15 +55,24 @@ private[kafka010] sealed trait ConsumerStrategy {
  * Specify a fixed collection of partitions.
  */
 private[kafka010] case class AssignStrategy(partitions: Array[TopicPartition])
-    extends ConsumerStrategy {
+    extends ConsumerStrategy with Logging {
   override def createConsumer(
       kafkaParams: ju.Map[String, Object]): Consumer[Array[Byte], Array[Byte]] = {
     val updatedKafkaParams = setAuthenticationConfigIfNeeded(kafkaParams)
+    excludeGroupId(updatedKafkaParams)
```

Comment:
I'd rather not manipulate passed map, as we did in setAuthenticationConfigIfNeeded.

Reply:
Now using KafkaConfigUpdater